server/public/styles.css

```diff
@@ -60,3 +75,18 @@ select {
 .white-space {
   white-space: pre-wrap;
 }
+
+.hidden {
+  display: none;
+}
```

Comment:
Bootstrap has display helpers.

https://getbootstrap.com/docs/4.5/utilities/display/#notation

Reply:
Didn't see that one! I noticed the invisible class but noticed that this left space as if the element was there